packages/3dom/src/test-helpers.ts

```diff
@@ -102,7 +102,7 @@ export class FakeModelKernel implements ModelKernel {
 }
 
 export const createFakeGLTF = () => {
-  const scene = new Scene();
+  const scene = new Group();
```

Comment:
I think I did that change here; or are you looking for something else?

Reply:
Sorry, you're right. LGTM.